include/rmm/device_buffer.hpp

```diff
@@ -392,4 +392,22 @@ class device_buffer {
       mr::get_default_resource()};  ///< The memory resource used to
                                     ///< allocate/deallocate device memory
 };
+
+/**--------------------------------------------------------------------------*
+ * @brief Copies rmm::device_buffer to a preallocated host buffer.
+ *-------------------------------------------------------------------------**/
+void copy_to_host(device_buffer& db, void* hb, cudaStream_t stream = 0) {
```

Comment:
```suggestion
void copy_to_host(device_buffer const& db, void* hb, cudaStream_t stream = 0) {
```

Reply:
I think this is was non-`const` because we were changing the stream. Though it sounds like Mark would like us not to do that. So it should be possible to make this `const` then.